addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The skeleton provided pom didn't work, so I tried different poms I have seen in other test bindings. The `<?xml` declaration should not be a problem though, it is an xml after all.